### PR TITLE
Fix task tags update on canvas cards [SCI-2875]

### PR DESCRIPTION
**Pull request by okriuchykhin**
https://biosistemika.atlassian.net/browse/SCI-2875

### CODE REVIEW

File: app/controllers/my_module_tags_controller.rb [hunk 1]
```diff
@@ -38,6 +34,26 @@ def index
...
   end
 
+  def canvas_index
+    experiment = Experiment.find(params[:id])
+    render_403 unless can_read_experiment?(experiment)
+    res = []
+    experiment.active_my_modules.each do |my_module|
+      res << {
+        id: my_module.id,
+        tags: render_to_string(
```
[the remark this answers is not in the document]
Comment (okriuchykhin) [reply app/controllers/my_module_tags_controller.rb]: renamed

File: app/views/experiments/canvas.html.erb [hunk 2]
```diff
@@ -45,7 +45,7 @@
     </div>
   </div>
 
-  <div id="canvas-container" data-project-id="<%= @project.id %>">
+  <div id="canvas-container" data-project-id="<%= @project.id %>" data-module-tags-url="<%= my_module_tags_experiment_path(@experiment) %>">
```
[the remark this answers is not in the document]
Comment (okriuchykhin) [reply app/views/experiments/canvas.html.erb]: added

**Comment by okriuchykhin** [comment]
> The tags do get updated, but not in the correct order - when you refresh the page, the order of the added tags is correct. I mean the order of tags on the task header, not in the model - over there the order is OK.

I couldn't reproduce this issue, from what I see we do not enforce any specific order on tags, so they are fetched in default order from DB(usually ordered by id)

**Comment by radiokills** [comment]
> The tags do get updated, but not in the correct order - when you refresh the page, the order of the added tags is correct. I mean the order of tags on the task header, not in the model - over there the order is OK.

I couldn't reproduce this,

**Pull request merged**